### ISSUE TITLE
Split cyto development into staging and production

### ISSUE THREAD

**Issue by daviddao** [cytoai/cyto#12]
We should have a staging product for immediate tests and development which we could host at `staging.cyto.ai` and a production-ready app at `production.cyto.ai` ~> `cyto.ai/app`

**Comment by 0x00b1** [comment]
@daviddao Yeah. For sure. However, for the moment, we prefer the faster iteration cycle from foregoing environments and promotions.
